File: Course-3/Mobile-dev/Introduction/Introduction_task/app/src/main/java/org/hse/android/MainActivity.java
package org.hse.android;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d("STRING", "Строчка с моим именем и фамилией: Ануфриев Павел");
        setContentView(R.layout.activity_main);
    }
}

File: Course-3/Mobile-dev/Stepik/5_repeat/5-1-3/app/src/main/java/com/example/company/myapplication/MainActivity.kt
package com.example.company.myapplication

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    fun handler0(v : View){
        if(number.text.toString().toIntOrNull() == null || systemOfCalculus.text.toString().toIntOrNull() == null
                || systemOfCalculus.text.toString().toInt() < 2 || systemOfCalculus.text.toString().toInt() > 36){
            convertResult.setText("Error")
        } else {
            convertResult.setText(number.text.toString().toBigInteger().toString(systemOfCalculus.text.toString().toInt()))
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        convertButton.setOnClickListener(this::handler0)
    }
}
